src/plugins/intel_npu/src/backend/src/zero_init.cpp

```diff
@@ -105,15 +105,14 @@ ZeroInitStructsHolder::ZeroInitStructsHolder() : log("NPUZeroInitStructsHolder",
     }
 
     // Query our graph extension version
-    uint32_t driverExtVersion = 0;
     std::string graphExtName;
-    std::tie(driverExtVersion, graphExtName) = queryDriverExtensionVersion(driver_handle);
+    std::tie(_driver_ext_version, graphExtName) = queryDriverExtensionVersion(driver_handle);
```

Comment:
We should choose some style guidelines, it feels a bit awkward to have `camelCase` and `snake_case` on the same line

Reply:
Yes, but I think I have to change more than this. In multiple places, we are using `camelCase` for local variables and `snake_case` for instance variables. I just used the same style as we have right now.  I will change in this function all variables to `snake_case`.